### PR TITLE
blog: embed funnel in your app

### PR DESCRIPTION
**Pull request by crawshaw**

### CODE REVIEW

File: data/blog/embedded-funnel.mdx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+authors: ['maisem']
+images: ['/images/libtailscale/social.png']
+summary: 'Tailscale Funnel is now available as a Go library you can embed directly in your programs.'
+---
+
+We recently released
+[Tailscale Funnel](https://tailscale.com/kb/1223/tailscale-funnel/)
+for testing, which lets you expose a locally running web server to the internet.
+
+Funnal can be especially useful when you are hosting services in a homelab, or
```
Comment (Xe) [inline data/blog/embedded-funnel.mdx:17]:
```suggestion
Funnel can be especially useful when you are hosting services in a homelab, or
```

File: data/blog/embedded-funnel.mdx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+developing against webhook APIs that need to POST back to your server.
+The standard way to use Funnel is to install Tailscale, run your server
+on localhost, and then configure Funnel to share that server.
+
+Today we are adding another way to use Funnel: compile it into your program.
+No need to install a separate Tailscale on the host.
```
Comment (Xe) [inline data/blog/embedded-funnel.mdx:23]:
```suggestion
When you do this, there is no need to install Tailscale on the host.
This allows Funnel to work in serverless environments such as Heroic or fly.io.
```

File: data/blog/embedded-funnel.mdx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+log.Fatal(http.Serve(ln, h))
+```
+
+See the full example on GitHub in
+[tsnet-funnel.go](https://github.com/tailscale/tailscale/blob/main/tsnet/example/tsnet-funnel/tsnet-funnel.go).
+
+After enabling HTTPS and Funnel in the Tailscale admin panel,
+you can generate an
+[auth key](https://tailscale.com/kb/1085/auth-keys/)
+and use it to run run this example:
```
Comment (Xe) [inline data/blog/embedded-funnel.mdx:54]:
```suggestion
and use it to run this example:
```

File: data/blog/embedded-funnel.mdx [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+```sh
+TS_AUTHKEY=<your-key> go run ./tsnet-funnel.go
+```
+
+Your local web server is now available on the internet under your
```
Comment (Xe) [inline data/blog/embedded-funnel.mdx:60]:
```suggestion
Your local web server will be available on the internet with your
```

**Review by Xe — APPROVED** [review]
Minor typo fixes, otherwise LGTM :shipit:

**Review by maisem — APPROVED** [review]

**Pull request merged**